src/bp-core/classes/trait-attachments.php

```diff
@@ -139,6 +139,7 @@ protected function upload_cover_from_file( $file ) {
 		}
 
 		return sprintf(
+			/* translators: %1$s: baseurl, %2$s: subdir, %3$s: basename. */
```

Comment:
```suggestion
```
Nothing is translated there, we don't need this comment imho.

Reply:
Good catch!